src/compiler/file_uploader/clear_file_and_info_actor.ts

```diff
@@ -0,0 +1,20 @@
+import { Actor, ActorSubclass } from '@dfinity/agent';
+import { createAuthenticatedAgent } from '../../../dfx';
+
+export async function createClearFileAndInfoActor(
+    canisterId: string
+): Promise<ActorSubclass> {
```

Comment:
Doesn't this return type remove useful information?

Reply:
done